services/web/server/src/simcore_service_webserver/login/handlers.py

```diff
@@ -162,6 +167,9 @@ async def logout(request: web.Request) -> web.Response:
         body = await request.json()
         client_session_id = body.get("client_session_id", None)
     await observer.emit("SIGNAL_USER_LOGOUT", user_id, client_session_id, request.app)
+    # Invalidate session
+    session = await get_session(request)
+    session.invalidate()
```

Comment:
By invalidating the session, you remove ALL sessions, therefore ``forget`` has no effect

Reply:
should get rid of it? or do we want to keep it anyway just in case we want to have some side-efffects?

Reply:
a test of a use case could reveal early that side-effects you are talking about.

Reply:
> a test of a use case could reveal early that side-effects you are talking about.

I don't understand this. I mean side effects inside ``forget`` like for example if we want to have some session information stored in the db or idk... informing other services or whatever. For the moment, nothing is being done